### PR TITLE
FIXED MANNY'S TEARS

### PR DESCRIPTION
**Pull request by dcoy**
@kungfuguapo  💭

### CODE REVIEW

File: mannsucvk.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+NO HE DOESN'T
+
```
Comment (kungfuguapo) [inline mannsucvk.txt:2]: But he does  tho

**Comment by kungfuguapo** [comment]
I'm a bit conflicted on approving this. Clearly Manny DOES suck but he has plenty of moments when he doesnt. Id be willing to approve this merge if you can first fix the wording to "Manny Sometimes Sucks"

**Comment by kungfuguapo** [comment]
For posterity the new code should read as

`MANNY SOMETIMES SUCKS`

Can we agree that this is a viable compromise?

Circle `Yes` or `No`.

K thx.

**Comment by dcoy** [comment]
YES @kungfuguapo

**Pull request closed without merging**